mobile/src/screens/auth/sign-in/index.tsx

```diff
@@ -1,5 +1,71 @@
-import { View } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+import { useState } from "react";
+import { StyleSheet, View } from "react-native";
+import { Button, Input } from "react-native-elements";
```

Comment:
We're going to use some UI library? Have you searched others?

Reply:
We gonna usehttps://nativebase.io